data/pom.xml

```diff
@@ -34,6 +34,7 @@
         <module>hdfs</module>
         <module>actian</module>
         <module>amazon-athena</module>
+        <module>amazon-simpleDB</module>
```

Comment:
make it like `amazon-simpledb` loose capital `DB`, it also means changing the directory name with similar semantics

Reply:
yeah this seems right. Let me make another commit here quickly.